pkg/where/match.go

```diff
@@ -85,6 +85,41 @@ func clearGlob(query string) string {
 	return query
 }
 
+func HasUnmatchedBrackets(query string) bool {
+	matchingBracket := map[rune]rune{
+		'}': '{',
+		']': '[',
+	}
+	stack := make([]rune, 0)
+
+	nodeHasUnmatched := func(query string) bool {
+		for _, c := range query {
+			if c == '{' || c == '[' {
+				stack = append(stack, c)
+			}
+			if c == '}' || c == ']' {
+				if len(stack) == 0 {
+					return true
+				}
+				if stack[len(stack)-1] == matchingBracket[c] {
```

Comment:
more readable:
```go
if stack[len(stack)-1] != matchingBracket[c] {
    return true
}

stack = stack[:len(stack)-1]
```

Reply:
Done in [38f3ea7](https://github.com/go-graphite/graphite-clickhouse/pull/284/commits/38f3ea7caf98ee0ed95e8e8b79b5d997164b460d)